app/src/main/java/com/github/libretube/ui/views/CustomExoPlayerView.kt

```diff
@@ -65,42 +76,68 @@ internal class CustomExoPlayerView(
         if (isControllerFullyVisible) hideController() else showController()
     }
 
-    private val doubleTouchListener = object : DoubleTapListener() {
-        override fun onDoubleClick() {
-            if (!doubleTapAllowed) return
-            val eventPositionPercentageX = xPos / width
-            when {
-                eventPositionPercentageX < 0.4 -> rewind()
-                eventPositionPercentageX > 0.6 -> forward()
-                else -> {
-                    player?.let { player ->
-                        if (player.isPlaying) {
-                            player.pause()
-                        } else {
-                            player.play()
-                        }
-                    }
+    private val playerGestureListner = object : PlayerGestureOptions {
```

Comment:
Please let the `CustomExoPlayerView` extend the `PlayerOptionsInterface` and then override the actions for the `CustomExoPlayerView` instead of creating an extra variable for it to keep the code easier to overview :)

Reply:
That sounds good. Implemented `PlayerGestureOptions` on `CustomExoPlayerView`.